src/renderer/components/RetypePasswordForm.tsx

```diff
@@ -7,16 +7,76 @@ import {
 } from "@material-ui/core";
 import { makeStyles } from "@material-ui/styles";
 import React, { useState, ChangeEvent, MouseEvent } from "react";
-import zxcvbn from "zxcvbn";
+import zxcvbn, { ZXCVBNFeedbackWarning } from "zxcvbn";
 import { RetypePassword } from "../../interfaces/i18n";
-import { useLocale } from "../i18n";
+import { T } from "@transifex/react";
+import { t } from "@transifex/native";
```

Comment:
What's different between `t` and `T`?

Reply:
`T` is a React component, and `t` is the underlying function for retrieving translated strings.

Reply:
Could we use only one type?

Reply:
I thought that would be impossible but after looking the code back, I think I could refactor to using a React variant only.